### ISSUE TITLE
Podcast promo - a11y - JAWS does not read out the text on the podcast promo

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#8784]
When testing the podcast promo with JAWS on IE11, on the oage and in storybook, the text content is not read out.

When using tab to navigate through the link elements, it reads the link as 'Russian heading level 3 link'

When using the arrow key to navigate through all elements it reads the following (each line represents one arrow press):

Region
Heading level 2 Russian
Heading level 3 link
Blank
Russian dash
[silence]
Blank
Russian
Region end

**Comment by LilyL0u** [comment]
Retested and I think this is because JAWS does not recognise the cyrillic alphabet. It reads the element text on the page that is in the latin alphabet, but not any in cyrillic. Not a problem with our code.

**Comment by LilyL0u** [comment]
Russia was the first country to put Robert Burns on a postage stamp

**Comment by gavinspence** [comment]
Changed the language on the a11y laptop to Russian and believe it reads out all of the elements.
